src/ui/about-respec.js

```diff
@@ -26,6 +26,9 @@ const localizationStrings = {
   nl: {
     about_respec: "Over",
   },
+  ja: {
+    about_respec: "これについて",
```

Comment:
#2687 will use `about_respec` in the respec pill menu item, so this will be shown as e.g. "これについて 2.4.10". Not sure it would be good enough, what do you think?

Reply:
Yes for this point.
Also I've gone through all translations, and I think this kind of items does not exist no more (other than this).